.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,14 +2,15 @@
 
 <!-- Please fill out the following: -->
 
-0. [ ] Project URL:
-1. [ ] Why should this project be added:
-2. [ ] End all descriptions with a full stop/period
-3. [ ] Entry is ordered alphabetically
-4. [ ] Appropriate icon(s) added if applicable (OSS, freeware)
+0. [ ] I have read the [Contribution Guidelines](https://github.com/iCHAIT/awesome-macOS/blob/master/.github/contributing.md)
+1. [ ] Project URL:
+2. [ ] Why should this project be added:
+3. [ ] End all descriptions with a full stop/period
+4. [ ] Entry is ordered alphabetically
+5. [ ] Appropriate icon(s) added if applicable (OSS, freeware)
```

Comment:
@herrbischoff Does it makes sense to keep the `reading the Contribution Guidelines` as the last point in the checklist since the project being suggested would be of higher importance relatively?

Looks great to me other than that, feel free to merge right away or after reordering the checklist :)

Reply:
will be merging, let's update this if needed

thx